README.md

```diff
@@ -582,6 +585,34 @@ $a->getValues(); // [0 => 'a', 1 => 'b', 2 => 'c']
 $a = A::create(['a', 'b', 'c']);
 $a->indexOf('b'); // 1
 ```
+### intersect
+
+> Associated with [array_intersect()](http://php.net/manual/en/function.array-intersect.php).
+
+``` php
+$a = A::create(['a', 'b', 'c']);
+$a->intersect(['b', 'c']);
+$a->toArray(); // [0 => 'b', 1 => 'c']
```

Comment:
As I understand, result should be `$a->toArray(); // [1 => 'b', 2 => 'c']` because this function preserve keys of first input array:
https://ideone.com/htVYO1